solidity/hardhat.config.ts

```diff
@@ -101,7 +101,7 @@ const config: HardhatUserConfig = {
       allowUnlimitedContractSize: process.env.TEST_USE_STUBS_TBTC === "true",
     },
     system_tests: {
-      url: "http://localhost:8545",
+      url: "http://127.0.0.1:8545",
```

Comment:
For some reason `http://localhost:8545` stopped working.
Possibly related to [this](https://stackoverflow.com/questions/68513645/error-connecting-to-localhost-after-npm-hardhat-run).